modules/misc/gtk.nix

```
@@ -97,9 +150,8 @@ in {
           type = types.path;
           default = "${config.home.homeDirectory}/.gtkrc-2.0";
           defaultText =
-            literalExpression ''"''${config.home.homeDirectory}/.gtkrc-2.0"'';
-          example =
-            literalExpression ''"''${config.xdg.configHome}/gtk-2.0/gtkrc"'';
+            literalExpression "${config.home.homeDirectory}/.gtkrc-2.0";
+          example = literalExpression "${config.xdg.configHome}/gtk-2.0/gtkrc";
```

This won't work when building the documentation.

```suggestion
            literalExpression ''""''${config.home.homeDirectory}/.gtkrc-2.0"'';
          example = literalExpression ''"''${config.xdg.configHome}/gtk-2.0/gtkrc"'';
```

---

I apologize. I didn't realize the purpose of escaping the double quotes at the time. It has been reverted now.